larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -268,6 +268,8 @@ const pandora::ParticleFlowObject *const pInputPfo)
     LArPfoHelper::GetTwoDClusterList(pInputPfo, clusterList);
     float diffWithStraightLineMean(0.f), maxFitGapLength(0.f), rmsSlidingLinearFit(0.f);
 
+	int nClustersUsed(0);
```

Comment:
Is the indentation actually non-standard here, or is it just the display in GitHub? It looks like 8 spaces per old "tab", rather than our usual 4. Also other indentation to check in this file, below.